src/wrappers.jl

```diff
@@ -176,6 +176,10 @@ function update!(
         )
     end
 
+    # Convert lower and upper bounds from Julia infinity to OSQP infinity
```

Comment:
you mean QPALM? :-)

Reply:
Ah sorry, I merged this just a second ago. Will fix in a sec.

Reply:
Ah in fact, it was your fault Lorenzo as this comment came directly from the setup wrapper :-)